### PR TITLE
Docs: Add a callout box explaining Tech Preview

### PR DESCRIPTION
**Pull request by BeenzSyed**
- Added a Note explaining what Tech Preview is
- Removed the Tech Preview tag on the top of the Overview page.

### CODE REVIEW

**Comment by eikenb** [comment]
Would it be possible to get this as a block on every page in the NIA area of the docs? So we don't need to add it to each page separately. Otherwise I think we should come up with a standard block to include in every page (perhaps this).

**Comment by lornasong** [comment]
@BeenzSyed just wanted to follow up regarding @-eikenb's question about if it's possible to add the block on every page? I could see users missing the note on the Overview page. I'm guessing it's not possible (or very difficult) and just want to confirm!

File: website/pages/docs/nia/installation/install.mdx [hunk 1]
```diff
@@ -8,7 +8,10 @@ description: >-
 
 # Install Consul-Terraform-Sync
 
-## Installing Consul-Terraform-Sync
+@include 'nia-tech-preview-note.mdx'
+
+Consul-Terraform-Sync is a daemon that runs alongside Consul, similar to other Consul ecosystem tools like Consul Template. Consul-Terraform-Sync is not included with the Consul binary and will need to be installed separately. First, we recommend installing Consul. For simplicity, we are using a local Consul agent in development mode which can be started with `consul agent -dev`. To quickly set up a local Consul agent, refer to the getting started [guide](https://learn.hashicorp.com/tutorials/consul/get-started-install). The Consul agent must be running to dynamically update network devices.
+
```
Comment (lornasong) [inline website/pages/docs/nia/installation/install.mdx:14]: just wanted to add some context, I had moved this section into the "Requirements" doc under a new "Install Consul" section. Here's the original conversation: https://github.com/hashicorp/consul/pull/8881#discussion_r501941993 and the new install consul: https://nia-docs-tech-preview--consul-docs-preview.netlify.app/docs/nia/installation/requirements#install-consul. the thinking was that this page would be the "install consul-terraform-sync" section vs. "general installations". let me know what you think though!

Comment (BeenzSyed) [reply website/pages/docs/nia/installation/install.mdx:14]: Yes, that makes sense! Sorry, I think I did this by mistake. Just to confirm, should I take this paragraph out?

Comment (lornasong) [reply website/pages/docs/nia/installation/install.mdx:14]: No worries! Yea, if you think moving the consul install content makes sense, you can remove this paragraph and I think it'll be all set

**Comment by BeenzSyed** [comment]
Closing this for now. I need to fix the automatic formatting happening on my side.

**Pull request closed without merging**